toolchain/check/convert.h

```diff
@@ -56,7 +56,8 @@ struct ConversionTarget {
 
 // Convert a value to another type and expression category.
 auto Convert(Context& context, SemIR::LocId loc_id, SemIR::InstId expr_id,
-             ConversionTarget target) -> SemIR::InstId;
+             ConversionTarget target,
+             SemIR::InstId vtable_id = SemIR::InstId::None) -> SemIR::InstId;
```

Comment:
Eventually, the vtable parameter here shouldn't be necessary (the vptr initialization should be handled by the initialization of a class of non-`partial` type, not the initialization of the base-most type). Please can you add a TODO somewhere to track this?

Reply:
Yeah, for sure - added the TODO here on `Convert` (since that seems to be the place that this awkwardness is most exposed) in 27493b718c9e995c72ef7406aa0e0b8644b189b9